ksqldb-functional-tests/src/test/resources/query-validation-tests/suppress.json

```diff
@@ -15,10 +92,222 @@
         {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 5},
         {"topic": "input_topic", "key": "k1", "value": {"COL1": "v1"},"timestamp": 0}
       ],
+      "outputs": [
+        {"topic": "OUTPUT", "key": "k1", "value": {"COUNT": 4},"window": {"start": 0, "end": 2, "type": "time"},"timestamp": 1},
+        {"topic": "OUTPUT", "key": "k1", "value": {"COUNT": 1},"window": {"start": 2, "end": 4, "type": "time"},"timestamp": 2}
+      ]
+    },
+    {
+      "name": "should handle null values",
```

Comment:
noting here in case the test doesn't come up later - what happens if we suppress with a tombstone? (e.g. the value coming in is entirely `null`, not a `null` column within the value)

Reply:
The `FinalResultsSuppressionBuilder` defaults to dropping tombstones, I will add a test to double check this

Reply:
@vvcephei Almog said that tombstones happen only on source tables, and since we are only suppressing on windowed streams, we don't have to worry about handling tombstones? What do you think

Reply:
Hey @nae701 and @agavra , a "tombstone" in this case means that the result of the windowed aggregation is `null`, not that the input record's value is `null`. I'm not sure if any of the available aggregations in KSQL can do this. If so, you can test it; if not, you don't have to worry about it.

For example (taken from the docs):
```
SELECT orderzip_code, TOPK(order_total, 5) FROM orders
  WINDOW TUMBLING (SIZE 1 HOUR) GROUP BY order_zipcode
  EMIT CHANGES;
```

In this case, the suppression would get a tombstone iff `TOPK(order_total, 5)` returns `null` (which I don't think it does, nor does COUNT, etc...).

Can you plug in a UDF for the aggregation function? If so, you can make it return nulls as desired and test the tombstone case.

Reply:
I thought about this some more, I don't think it's possible for ksql to produce a tombstone in this case. Even if `TOPK` would return `null`, we still don't get a tombstone - we just get a `null` value for the column (probably `KSQL_COL_2` or whatever the generated name would be in this case